AdyenSession/AdyenSession.swift

```diff
@@ -64,13 +66,16 @@ public final class Session: SessionProtocol {
     /// The presentation delegate.
     public private(set) weak var presentationDelegate: PresentationDelegate?
     
-    /// Initializes an instance of `Session` asynchronously.
+    /// The delegate object.
+    public weak var delegate: AdyenSessionDelegate?
```

Comment:
Question: We wanna force the merchant to always set a session delegate right? Cause in that case passing it as a non optional in line 76 makes sense